data/ui/ButtonDialog.ui

```diff
@@ -250,4 +251,31 @@
       <action-widget response="apply" default="true">apply</action-widget>
     </action-widgets>
   </template>
+  <object class="GtkListBoxRow" id="row_left_hand_mode">
+    <property name="width_request">100</property>
+    <property name="visible">True</property>
+    <property name="can_focus">True</property>
+    <child>
+      <object class="GtkBox">
+        <property name="visible">True</property>
+        <property name="can_focus">False</property>
+        <property name="margin_top">4</property>
+        <property name="margin_bottom">4</property>
+        <child>
+          <object class="GtkLabel">
+            <property name="visible">True</property>
+            <property name="can_focus">False</property>
+            <property name="halign">start</property>
+            <property name="label" translatable="yes">Switch the left- and right mouse buttons</property>
```

Comment:
I think this should be "Swap the left and right mouse buttons", see https://english.stackexchange.com/questions/22954/is-there-any-difference-between-switch-and-swap for a (non-authoritative) explanation between switch and swap.

But what I'm not happy with here is that if I click on the mouse button, the only option I have now is to swap and my only recourse is cancel. This is a bit confusing, shouldn't there be an entry for the current option too? i.e. right-handed?

I'm not happy with it anyway, it's not expressive enough, but I don't know how to make it better with this design. I'm starting to think it may be worth having a different dialog style here and for the right button that only offers left/right click and possibly a little graphic or explanation. Basically what the Mouse&Touchpad panel in the control center does.